src/components/molecules/ProjectCard.js

```diff
@@ -37,98 +39,97 @@ class ProjectCard extends Component {
+        if (projects) {
+            return (
+                <div className={partial ? "container" : "container project-cards-full"}
+                >
+                    {(!partial) ? (
+                        <SearchBox projects={this.props.projects} filters={this.state.filters} onTagsUpdate={this.handleFilterUpdate}/>
+                    ) : (
+                        null
+                    )}
+                    <div className="row justify-content-center">
+                        {projects.map((project, i) => {
+                            if (!partial || i < this.state.limit) {
+                                return (
+                                    <Card 
+                                        key={project._id}
+                                        user={this.props.user}
+                                        project={project}
+                                        onClick={() => this.props.history.push('/project/view/' + project._id)}
+                                        onFollow={this.handleClick.bind(this, project._id)}
+                                    />
+                                );
+                            } else {
+                                return null
+                            }
+                        })}
+                    </div>
+                    <div className="text-center">
+                        {(partial) ? 
+                            (<Button label="All projects" redirect="/project/view/"/>) :
+                            (<Button label="Back to main" redirect="/" />)
+                        }
+                    </div>
+                </div>
+            );
         } else {
-        return <Loader />
+            return <Loader />
         }
-    }
-    handleTagsUpdate = (filterArray) => {
-        this.props.onFilterUpdate(filterArray);
-    }
-    render() {
-        const partial = Boolean(this.props.limit);
-        return (
-            <div className={(partial) ? 
-                ("container") : 
-                ("container project-cards-full")}
-            >
-                {(!partial) ? (
-                    <SearchBox projects={this.props.projects} filters={this.props.filters} onTagsUpdate={this.handleTagsUpdate}/>
-                ) : (
-                    null
-                )}
-                <div className="row justify-content-center">
-                    {this.setProjects()}
-                </div>
-                <div className="text-center">
-                    {(partial) ? 
-                        (<Button label="All projects" redirect="/project/view/"/>) :
-                        (<Button label="Back to main" redirect="/" />)
-                    }
-                </div>
-            </div>
-        );
+        
     }
 }
 
+const Card = ({user, project, onClick, onFollow}) => (
+    <div className="col-md-3 card"
+        onClick={onClick}
+        key={project._id}
+        id={project._id}>
+        {user ? (
```

Comment:
Currently the follow button is showing on all projects. Can you add in a condition where if the user has created that project that the icon does not show for them. Since they will be doing the updates, I don't think they need to be alerted about it.